gemrb/plugins/2DAImporter/2DAImporter.cpp

```diff
@@ -25,81 +25,55 @@
 
 using namespace GemRB;
 
-#define SIGNLENGTH 256      //if a 2da has longer default value, change this
-
 static bool StringCompKey(const std::string& str, TableMgr::key_t key)
 {
 	return stricmp(str.c_str(), key.c_str()) == 0;
 }
 
-p2DAImporter::p2DAImporter() noexcept
-{
-	colNames.reserve(10);
-	rowNames.reserve(10);
-	rows.reserve(10);
-}
-
```

Comment:
Why did you remove these? They were added to reduce the amount of reallocations and helped significantly when you looked at a heap profile.

Reply:
they've been added to `p2DAImporter::Open`